packages/fxa-auth-server/test/mail_helper.js

```diff
@@ -40,7 +40,7 @@ async function printMatchingKeys(startUp = false) {
       }
     }
   } catch (error) {
-    console.error('Failed to retrieve keys:', error);
+    // do nothing
```

Comment:
This was causing lots of errors to show in console even when no codes were expected to be emitted.